### PR TITLE
EOS-22875: Resolved sorting/mapping discrepancy for server nw_port resource.

### PR DESCRIPTION
**Pull request by satish-seagate**
Signed-off-by: Satish Darade <satish.darade@seagate.com>

## Coding:

* [x] Did you follow coding standard and verified it with flake8? 

* [x] Did your address all Codacy issues?

## Testing:

* [x] Unittest updated and all unittests are passing?

* [x] Sanity/Self tests are updated (if applicable)?

* [x] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [x] Did you add Jira number to the PR?

* [x] DCO and cla-signed?

### CODE REVIEW

File: low-level/solution/lr2/server/health.py [hunk 1]
```diff
@@ -606,10 +607,15 @@ def get_nw_ports_info(self):
             )
             self.set_health_data(nic_info, health_status, description=desc,
                                  specifics=[specifics])
-            network_cable_data.append(nic_info)
+            if interface == 'lo':
```
Comment (cdeshmukh) [inline low-level/solution/lr2/server/health.py:610]: add comment plz, on why we are doing this

Comment (cdeshmukh) [inline low-level/solution/lr2/server/health.py:610]: add comment plz, on why we are doing this

**Review by cdeshmukh — APPROVED** [review]

File: low-level/solution/lr2/server/health.py (hunk 1 above)
Comment (satish-seagate) [reply low-level/solution/lr2/server/health.py:612]: Done

Comment (satish-seagate) [reply low-level/solution/lr2/server/health.py:612]: Done

**Review by thavanathan — APPROVED** [review]

**Review by mariyappanp — APPROVED** [review]

**Review by jjxsg — APPROVED** [review]

**Comment by cdeshmukh** [comment]
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/SSPL/541/
deployment had passed

**Pull request merged**